TabletopTweaks/NewContent/ContentAdder.cs

```diff
@@ -73,6 +73,9 @@ static class BlueprintsCache_Init_Patch {
                 Feats.DervishDance.AddDervishDance();
 
                 AlternateCapstones.MasterfulTalent.AddMasterfulTalent();
+
+                //custom
+                Custom.VitalStrikeFix.FixVitalStrike();
```

Comment:
this can be pulled out and executed instead in TabletopTweaks.Bugfixes.Features.Feats